pages/common/chronic.md

```diff
@@ -0,0 +1,16 @@
+# chronic
+
+> Display `stdout` and `stderr` of a command if and only if it fails.
+> More information: <https://joeyh.name/code/moreutils/>.
+
+- Display `stdout` and `stderr` of the specified command if and only if it produces a non-zero exit code or crashes:
+
+`chronic {{command options ...}}`
+
+- Display `stdout` and `stderr` of the specified command if and only if it produces a non-empty `std[e]rr`:
```

Comment:
I think the brackets of `std[e]rr` will be rendered as actual characters by the clients

Reply:
Okay, changed it